packages/graphback-schema/src/visitor.ts

```diff
@@ -0,0 +1,7 @@
+import { ObjectTypeDefinitionNode } from 'graphql';
+
+export const visitor = {
+  ObjectTypeDefinition: (node: ObjectTypeDefinitionNode) => {
```

Comment:
I think we miss field definition here. 
Without that we do not have access to fields

Reply:
generally add more definitions here and see how we can process things